usage/installer/README.md

```diff
@@ -0,0 +1,48 @@
+# SeaClouds
+
+## Deploy SeaClouds
+A deployment of SeaClouds can be launched using Apache Brooklyn. We currently support deployments against Bring Your 
+Own Nodes (BYON) and to all the IaaS provider supported by [Apache jclouds](http://jclouds.org).
+
+# Deploy SeaClouds platform
+
+## Launching SeaClouds on the cloud
+
+```bash
+cd installer/target/seaclouds-installer-dist/seaclouds-installer
```

Comment:
`mvn clean install` is required before (even if obvious!) 
Also, the path should be `cd usage/installer/target/seaclouds-installer-dist/seaclouds-installer`.